pkgs/by-name/pd/pdfannots2json/package.nix

```
@@ -0,0 +1,22 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
```

```suggestion
buildGoModule {
```

`rec` might not be necessary

---

Ok, thanks for the review.